chromatic_fitting/models/combined.py

```diff
@@ -422,6 +423,18 @@ def setup_lightcurves(self, store_models: bool = False, **kw):
                         f"{self.name}_model", self.every_light_curve[f"wavelength_{i}"]
                     )
 
+    def sample(
+        self,
+        **kw,
+    ):
+        for m in self._chromatic_models.values():
+            if isinstance(m, EclipseModel):
+                EclipseModel.sample(self, **kw)
+                # "sampling_method" not in kw.keys():
+                # LightcurveModel.sample(self, sampling_method=sample_ext, **kw)
+            else:
+                LightcurveModel.sample(self, **kw)
+
```

Comment:
combined models that have an eclipse will have to use pymc3_ext.sample for now...